src/components/views/login/PasswordLogin.js

```diff
@@ -199,24 +200,24 @@ class PasswordLogin extends React.Component {
             <div>
                 <form onSubmit={this.onSubmitForm}>
                 <div className="mx_Login_type_container">
-                    <label className="mx_Login_type_label">I want to sign in with my</label>
+                    <label className="mx_Login_type_label">{ counterpart.translate('I want to sign in with') }</label>
                     <Dropdown
                         className="mx_Login_type_dropdown"
                         value={this.state.loginType}
                         onOptionChange={this.onLoginTypeChange}>
-                            <span key={PasswordLogin.LOGIN_FIELD_MXID}>Matrix ID</span>
-                            <span key={PasswordLogin.LOGIN_FIELD_EMAIL}>Email Address</span>
-                            <span key={PasswordLogin.LOGIN_FIELD_PHONE}>Phone</span>
+                            <span key={PasswordLogin.LOGIN_FIELD_MXID}>{ counterpart.translate('my Matrix ID') }</span>
```

Comment:
I would move "my" to the previous string (where it was before) as it fits to the other options as well. And it avoids confusion because "my" is just there in combination with Matrix ID